src/main/java/net/masterthought/cucumber/ReportParser.java

```diff
@@ -97,4 +100,33 @@ private void setMetadata(Feature[] features, String jsonFile, int jsonFileNo) {
             feature.setMetaData(jsonFile, jsonFileNo, configuration);
         }
     }
+
+    /**
+     * Parses passed properties files and adds metadata to overview-features page.
```

Comment:
it's all about classification, can we use documentation and code which matches this keyword as well?

Reply:
Yeah no problem I'll update

Reply:
should I rename parsePropertiesFiles to parseClassificationsFiles?